pkg/ccl/changefeedccl/sink_kafka_v2.go

```diff
@@ -419,14 +424,17 @@ func buildKgoConfig(
 			}
 			tlsCfg.Certificates = []tls.Certificate{cert}
 		}
-		opts = append(opts, kgo.DialTLSConfig(tlsCfg))
+		dialer := &net.Dialer{Timeout: 10 * time.Second}
+		opts = append(opts, kgo.Dialer(netMetrics.WrapTLS(dialer.DialContext, tlsCfg, "kafka_v2")))
```

Comment:
```suggestion
		dialer := &net.Dialer{Timeout: 10 * time.Second}
		tlsDialer := &tls.Dialer{NetDialer: dialer, Config: tlsCfg}
		opts = append(opts, kgo.Dialer(netMetrics.Wrap(tlsDialer.DialContext, "kafka")))
```

I haven't tested this yet, but would this work as an alternative to needing the `WrapTLS` function?

Reply:
I will try that on the kafka_v2 commit. It would be nice if it does.